packages/design-system-tokens/src/css/translate.ts

```diff
@@ -149,6 +149,66 @@ export function tokensToCssProperties(
   return vars.join('\n');
 }
 
+/**
+ * Unlike CSS Custom Properties, Sass variables can't be used before they're defined. In
+ * order to avoid compilation errors in Sass, we need to enforce an order of declaration
+ * and then use, which may not be the natural order we get from the tokens.
+ */
+export function enforceSassVariableOrder(originalDeclarations: string[]): string[] {
+  // Matches the variable being declared (first group) and any variables being referenced (second group)
+  const declarationRegex = /(\$[\w\-_]+):\s*(\$[\w\-_]+)?.*;/;
+  const getVarNames = (line) => line.match(declarationRegex)?.slice(1) || [];
```

Comment:
Jenkins [run](https://ci.backends.cms.gov/wds/job/design-system/job/tests/job/PR-3232/1/console) notes a typescript error here (`src/css/translate.ts(160,24): error TS7006: Parameter 'line' implicitly has an 'any' type.`)

Reply:
Strange that I didn't get that locally. Fixed it